server.js

```diff
@@ -152,25 +177,51 @@ app.get('/broad', (req, res) => {
 
 // LOGIN RELATED INFORMATION
 
-app.get('/isLoggedIn', (req, res) => {
-  res.send('hitting server!!!!');
-});
-
 app.post('/login', (req, res) => {
-  // TODO: Senai do some stuff here
-  console.log('server post login endpoint');
-  console.log(req.body, 'rrreeeqqqq......bbbbooooddddyyyy');
-  // helpers.loginUser(req, res);
-  // helpers.createUser(req, res);
-  res.send(req.body);
-  // res.send('logged in');
+  const userInfo = req.body;
+  console.log(userInfo, '88888888888888888');
+  dbHelpers.findUserLogin(userInfo)
+    .then((user) => {
+      console.log('uuuuuuuussssssseeeeeeerrrrrrrr in server /login', user);
+      if (user === null) {
+        res.send(`Sorry ${userInfo.email}, we can not find you, try signing up 😊`);
+      } else if (user.dataValues.password === userInfo.password) {
+        console.log('PASSWORDIS A MATCH!!!!!!!!');
+        res.send('success');
+      } else {
+        res.send(`Sorry ${userInfo.email}, wrong passord`);
+      }
+      // res.send('success');
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+
+  // helpers.loginUser(req)
+  //   .then((user) => {
+  //     console.log('userFound', user);
+  //     console.warn('user found');
+  //     res.send('User Found');
+  //   })
+  //   .catch((error) => {
+  //     console.warn(error);
+  //   });
 });
 
-app.post('/signUp', (req, res) => {
-  console.log('signUp user fired');
-  // console.log('user: ', user);
-  console.log(req.body);
-  res.send(req.body);
+app.post('/signup', (req, res) => {
+  const userObject = req.body;
+  dbHelpers.findUserSignup(userObject)
+    .then((response) => {
+      console.log('server /signup #############', response);
+      if (response === '1') {
```

Comment:
what is '1' and '2' are these testing values or do are they a ref to something?

Reply:
#1 means the user signing up has a valid email address (an email address that is not already in the database)

#2 means that the email the user tries signing up with already exists in the database. If you look in the if block you can see the response to the client are different.

I definitely need these, they are not for testing. The is the response I am sending form the server. I can make them true and false if you prefer.